csuntechlab/checkpoint
# Check 138 - Create login page with authentication for Admins

# Description
- Added Login view
- Added Axios + Vuelidate
- Added Form validation
- Added Jest test to check if view exist
- Renamed api/login error message (email.required) to remove ambiguity
- Hits /api/login to login user; should return access_token

## Type of change

Please delete options that are not relevant.

# How Has This Been Tested?

- Run `yarn` to install new node_modules
- Spin up environment `docker-compose up -d`
- Navigate to [http://localhost:8080/#/login](http://localhost:8080/#/login)
- Enter a valid user and password; then sign in. 
- Console log should return access_token


# Checklist:
